actioncable/lib/action_cable/test_helper.rb

```diff
@@ -0,0 +1,68 @@
+module ActionCable
+  # Provides helper methods for testing Action Cable broadcasting
+  module TestHelper
+    extend ActiveSupport::Concern
+
+    included do
+      def before_setup # :nodoc:
+        server = ActionCable.server
+        test_adapter = ActionCable::SubscriptionAdapter::Test.new(server)
+
+        @old_cable_adapter = server.pubsub
+
+        server.instance_variable_set(:@pubsub, test_adapter)
+        super
+      end
+
+      def after_teardown # :nodoc:
+        super
+        ActionCable.server.instance_variable_set(:@pubsub, @old_cable_adapter)
+      end
+
+      def assert_transmissions(number, channel)
+        if block_given?
```

Comment:
No, we don't need it here, it's checked implicitly. This is how `block_given?` works.